src/codeflare_sdk/templates/base-template.yaml

```diff
@@ -69,7 +69,7 @@ spec:
         containers:
         # The Ray head pod
         - name: ray-head
-          image: quay.io/modh/ray@sha256:8b9e0efa3ae0e1862d66c02ffa82b80830334a5f8af12deb96d4f2f8babce5fe
+          image: quay.io/modh/ray:2.35.0-py39-cu121-b87d9a71c34c0b79c0e7aa9d4245fe7f0065f774
```

Comment:
Shouldn't we use the SHA generated for the tag to get disconnected environment working?